### ISSUE TITLE
Problem when setting sales channel

### ISSUE THREAD

**Issue by kingsaj** [CPIGroup/phpAmazonMWS#42]
When I set

$amz->setShowSalesChannel(true); the request seems to fail.

If i comment out that line, I get a response from Amazon fine.

The error I get back is below;

<Error>
   <Type>Sender</Type>
   <Code>InvalidParameterValue</Code>
   <Message>Invalid query string provided - SellerId={My Seller ID}&amp;AWSAccessKeyId={My AWSAccessKeyId}&amp;SignatureVersion=2&amp;SignatureMethod=HmacSHA256&amp;Version=2009-01-01&amp;Action=RequestReport&amp;ReportType=_GET_MERCHANT_LISTINGS_DATA_&amp;ReportOptions=ShowSalesChannel=true&amp;Timestamp=2015-05-09T19%3A57%3A53%2B0000&amp;Signature=TraU0pEELHRR1a2QF6wTBfM3ET4k7H4ZCu46c9OY0eo%3D is not valid; the value of a query string parameter may not contain a '=' delimiter</Message>
</Error>

**Comment by Peardian** [comment]
I see what the problem is. It's the "`ReportOptions=ShowSalesChannel=true`" part. The version of the documentation I used made it seem like this is how it is supposed to be sent. I checked again, and they have since updated the docs to indicate it should say "`ReportOptions=ShowSalesChannel%3Dtrue`" instead.

I'll make a fix for it when I am able to. In the meantime, I hope this info solves your problem.
